pages/common/kubectl.md

```diff
@@ -29,3 +29,7 @@
 - Print the address of the master and cluster services:
 
 `kubectl cluster-info`
+
+- Display an explanation of the fields shown:
```

Comment:
What about `Display an explanation of the specific field:` or `Display an explanation of the specific resource's field:`